src/classes/CRLP_RollupProcessor.cls

```diff
@@ -488,6 +488,15 @@ public inherited sharing class CRLP_RollupProcessor {
         // has changed or not.
         Id parentId = parent.Id;
 
+        Boolean hasSkewFlag = parent.getSObjectType() == Account.SObjectType || parent.getSObjectType() == Contact.SObjectType;
```

Comment:
For performance reasons, some of this can be moved into a method that is set a single time. Rollups are always run for the same object in the same context. So, instead of doing this compare and the `isSet()` call 200 times in an execution context (assuming a batch size of 200 times), we can do it just once the first time the method is called and re-use the result for all subsequent calls. In the below, I figure this method can call `getSkewModeValues(parent)` at the top to handle setting the values if needed. I also renamed some vars here.

```
private final String qualifiedSkewField;
private Boolean useForceSkewModeField;
private Boolean isSetSkewModeField;

private void getSkewModeValues(SObject parent) {
    if (qualifiedSkewModeField == null) {
        this.useForceSkewModeField = (parent.getSObjectType() == Account.SObjectType || parent.getSObjectType() == Contact.SObjectType);
        this.isSetSkewModeField = parent.isSet(qualifiedSkewField);
        this.qualifiedSkewField = UTIL_Namespace.StrTokenNSPrefix('CustomizableRollups_UseSkewMode__c');
    }
}
```